composer.json

```diff
@@ -24,14 +24,15 @@
         "symfony/event-dispatcher": "4.4.*|^5.0",
         "symfony/http-foundation": "4.4.*|^5.0",
         "symfony/http-kernel": "4.4.*|^5.0",
-        "symfony/options-resolver": "^5.0"
+        "symfony/options-resolver": "4.4.*|^5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "9.4.*",
+        "phpunit/phpunit": "9.5.13",
```

Comment:
I think it's not a good practice to fix specific version of packages in composer.json for open-source bundle.

May be you can simply specify `9.5.*`?